File: auth-service/src/main/java/io/simakkoi9/authservice/model/dto/security/response/DriverResponse.java
package io.simakkoi9.authservice.model.dto.security.response;

import io.simakkoi9.authservice.model.Gender;
import java.time.LocalDateTime;

public record DriverResponse(
        Long id,
        String name,
        String email,
        String phone,
        Gender gender,
        Long carId,
        LocalDateTime createdAt
) {}

File: driver-service/src/main/kotlin/io/simakkoi9/driverservice/model/dto/rest/driver/response/DriverResponse.kt
package io.simakkoi9.driverservice.model.dto.rest.driver.response

import io.simakkoi9.driverservice.model.entity.Gender
import java.io.Serializable
import java.time.LocalDateTime

data class DriverResponse(
    val id: Long,

    val name: String,

    val email: String,

    val phone: String,

    val gender: Gender,

    val carId: Long?,

    val createdAt: LocalDateTime
) : Serializable